### ISSUE TITLE
Selecting .WebM file crashes Godot

### ISSUE THREAD

**Issue by Dillybob92** [godotengine/godot#7220]
**Operating system or device - Godot version:** 2,2 alpha:
http://fixnum.org/godot/16_11_30__11_27_45/tools/godot.windows.tools.64.exe

**Issue description** (what happened, and what was expected):
Basically, if you add a `VideoPlayer` node, select WebM:
![image](https://cloud.githubusercontent.com/assets/10387670/20784728/fa273d06-b750-11e6-9087-86ff35d37c34.png)

 then load a .webm file Godot will poop.

@zaps166 Might be interested in this as he's a WebM specialist atm

I cannot get any other errors from the console however but will try to isolate this more if requested.
![image](https://cloud.githubusercontent.com/assets/10387670/20784647/88d77ba2-b750-11e6-9101-ea3ee4c2df89.png)

```
Problem signature:
  Problem Event Name:	APPCRASH
  Application Name:	godot.windows.tools.64 (1).exe
  Application Version:	2.2.0.0
  Application Timestamp:	a140a138
  Fault Module Name:	godot.windows.tools.64 (1).exe
  Fault Module Version:	2.2.0.0
  Fault Module Timestamp:	a140a138
  Exception Code:	c0000005
  Exception Offset:	0000000000269812
  OS Version:	6.1.7601.2.1.0.256.48
  Locale ID:	1033
  Additional Information 1:	781e
  Additional Information 2:	781e5a85ab2727cddc2ddaf44755c6ef
  Additional Information 3:	d5a4
  Additional Information 4:	d5a4b8627b171ec0cc3b651e1bd1c7ca
```

This error is just a generic Windows 7 crashing error I think, nothing really helpful :(

My operating system:
Windows 7, 64-bit Professional

Edit: Happens under 32 and 64 bit versions if that helps

Edit2: Oh, here is the webm if you need:
https://cdn.discordapp.com/attachments/105137795625979904/253780319063244801/headerbackground.webm

**Comment by zaps166** [comment]
I'm sure that I've tested video without audio stream, but maybe it has been broken after PR updates.

Please test: #7221 :)

**Comment by akien-mga** [comment]
Assumed fixed by #7221.
